pom.xml

```diff
@@ -61,6 +61,7 @@
     <semverVersion>0.9.0</semverVersion>
     <atlasVersion>1.0.1</atlasVersion>
     <pncVersion>2.0.0-SNAPSHOT</pncVersion>
+    <tagSuffix/>
```

Comment:
is this necessary?

Reply:
Tried it on my mock project and you are right, you don't need to have it explicitly declared. OTH I'd like to keep it there in <properties> where all available ones are declared.